src/python/bot/tasks/fuzz_task.py

```diff
@@ -1380,6 +1393,13 @@ def execute_task(fuzzer_name, job_type):
   thread_delay = environment.get_value('THREAD_DELAY')
   thread_error_occurred = False
 
+  # TODO: remove environment variable once refactor is complete
```

Comment:
Instead of having an if here, move everything into the get_fuzz_strategy_distribution function (and rename it to something more appropriate for that case).